tests/system_tests_multicast.py

```diff
@@ -506,26 +634,30 @@ def __init__(self, config, count, body, topic=None, **handler_kwargs):
 
         # per receiver
         self.c_received = {}
+        self.received_msgs = {}
 
         # count per outcome
         self.n_outcomes = {}
 
         self.error = None
         self.timers = []
         self.reactor = None
+        self.done_timer = None
 
     def done(self):
         # stop the reactor and clean up the test
         for t in self.timers:
```

Comment:
Was wondering why there needs to be a list of timers in `self.timers`
Can it just be self.timer ? I noticed the the `len(self.timers)` is always one

Reply:
I had on open question about this myself.

I notice that `MulticastBase.on_start()` has code to append to the timers list.
https://github.com/skupperproject/skupper-router/blob/main/tests/system_tests_multicast.py#L541

I was not sure if the result of `self.reactor.schedule(TIMEOUT, TestTimeout(self))` can/should be mixed with a threading.Timer.

If you prefer that I remove self.done_timer and append the 'done' timer to self.timers, just let me know.  I'll be curious to hear your preference...

Reply:
Removed done_timer.
Removed `import threading`.
Added poll_timer.